pkg/kubectl/cmd/apply_test.go

```diff
@@ -279,6 +296,85 @@ func TestApplyObject(t *testing.T) {
 	}
 }
 
+// TestApplyTPR uses apply commad with ThirdPartyResources.
+func TestApplyTPR(t *testing.T) {
```

Comment:
Submission of the ThirdPartyResource is not the important part to test. Submission of the kind that is not compiled in (e.g. A "Foo" object in group "example.com" and version "v1") multiple times (once to create and once to patch/apply)